### DISCUSSION TITLE
newbie questions about AP + hardware

### DISCUSSION THREAD

**Discussion by fugata** [davesteele/comitup#158] [Q&A]
Hello,
I came cross comitup while reading the article 

https://www.cnx-software.com/2021/03/26/customize-raspberry-pi-os-raspberry-pi-imager-v1-6/

This package sounds like something I've always wanted to use -- great!  I'm a matlab type of person and knows very little about networks, so apologies in advanced for some elementary questions.

1) Here's one of my usage scenario:  I'm working somewhere with no wifi/no secured wifi, and I need to connect a linux-based devices (e.g. ssh into my laptop from my android tablet, file severs etc).  I would like to e.g. use a headless raspberry pi to create access points so these devices (3-4 max) would talk to each other.  Would comitup takes care of the setting up of the acess point on the RPi?

2) Is the comitup package targeted for RPi only, or does it work for other single board arm computers that run some form of ubuntu/debian?  Reason:  I would like to use one of these SBC as a travel router/repeater/AP.   I'm hoping that comitup would help simplifying setups with captive portal.   RPi unfortunately does not support hardware crytpo so vpn speed would take quie a hit, but there are SBC with such supports.

Apologies again if I misunderstood the goal of this project, and thanks for you help!

**Comment by davesteele** [comment]
Multiple devices connected to a Comitup HOTSPOT should be able to communicate with each other. If there are multiple Comitup devices, and no upstream, pick one to serve as the host. I haven't tested this.
It should run on any deb-based system, but there is finagling to be done, and I've done minimal testing (and none recently) for anything other than Raspberry Pi OS. See the wiki for installations issues and WiFi hardware restrictions.
